### PR TITLE
CP-4 Add new functionality

### PR DESCRIPTION
**Pull request by githubTestUser21**
This functionality is supper fancy and you won't believe how cool it is. 🖖

### CODE REVIEW

File: component/input/index.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 function shortName() {
-    return (1 + 1 ) * 5
+    return (IDoSomethingSuperFancyHere(1, 4)) * 5
```
Comment (githubTestUser20) [inline component/input/index.js]: Methods names start with lowercase

File: component/input/index.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+}
+
+function IDoSomethingSuperFancyHere(x, y) {
+    return (y + x ) * 5
```
Comment (githubTestUser20) [inline component/input/index.js]: A `;` is missing at the end of this line.

**Review by githubTestUser20 — CHANGES_REQUESTED** [review]

File: component/input/index.js [hunk 3]
```diff
@@ -1,5 +1,9 @@
 function shortName() {
-    return (1 + 1 ) * 5 * 5;
+    return (IDoSomethingSuperFancyHere(1, 4)) * 5 * 5;
```
Comment (githubTestUser20) [inline component/input/index.js]: Please the function name should start with lowerCase

**Review by githubTestUser20 — CHANGES_REQUESTED** [review]
Please do the requested changes.

**Review by githubTestUser20 — APPROVED** [review]

File: component/input/index.js [hunk 4]
```diff
@@ -1,5 +1,9 @@
 function shortName() {
-    return (1 + 1 ) * 5 * 5;
+    return (iDoSomethingSuperFancyHere(1, 4)) * 5 * 5;
+}
+
+function iDoSomethingSuperFancyHere(x, y) {
```
Comment (githubTestUser20) [inline component/input/index.js]: I'd like some indentation here, but, don't do int now.

Comment (githubTestUser21) [reply component/input/index.js]: Ok, next time I'll do it.

File: component/input/index.js [hunk 5]
```diff
@@ -1,5 +1,9 @@
 function shortName() {
-    return (1 + 1 ) * 5 * 5;
+    return (iDoSomethingSuperFancyHere(1, 4)) * 5 * 5;
```
Comment (githubTestUser20) [inline component/input/index.js:2]: I just realised the 5 should be 5.1

**Review by githubTestUser20 — DISMISSED** [review]

File: component/input/index.js (hunk 5 above)
Comment (githubTestUser21) [reply component/input/index.js:2]: No, you are wrong. It has to be `5`.

Comment (githubTestUser20) [reply component/input/index.js:2]: Ok, you are right. I'll dismiss the review.

**Review by githubTestUser20 — APPROVED** [review]
Ok for me, after double check. I'll approve it.

**Review by githubTestUser20 — COMMENTED** [review]
Let's do a meeting about this changes.

File: component/input/index.js (hunk 4 above)
Comment (githubTestUser20) [inline component/input/index.js]: I just noticed the variables are to small.

**Review by githubTestUser20 — CHANGES_REQUESTED** [review]

**Review by githubTestUser20 — APPROVED** [review]

File: component/input/index.js [hunk 6]
```diff
@@ -1,5 +1,9 @@
...
+}
+
+function iDoSomethingSuperFancyHere(totalA, totalB) {
+    return (totalA + totalB ) * 5;
```
Comment (githubTestUser20) [inline component/input/index.js:6]: The number `5` is risky, but Ok !.

**Pull request merged**